src/ethernet/RsRtspClient.cpp

```diff
@@ -185,30 +185,34 @@ int RsRTSPClient::stop()
 
 int RsRTSPClient::close()
 {
-    unsigned res = this->sendTeardownCommand(*this->m_scs.m_session, this->continueAfterTEARDOWN);
-    // wait for continueAfterTEARDOWN to finish
-    std::unique_lock<std::mutex> lck(m_commandMtx);
-    m_cv.wait_for(lck, std::chrono::seconds(RTSP_CLIENT_COMMANDS_TIMEOUT_SEC), [this] { return m_commandDone; });
-    // for the next command
-    if (!m_commandDone)
     {
-        RsRtspReturnValue err = {RsRtspReturnCode::ERROR_TIME_OUT, "client time out"};
-        throw std::runtime_error(format_error_msg(__FUNCTION__, err));
-    }
-    m_commandDone = false;
+        unsigned res = this->sendTeardownCommand(*this->m_scs.m_session, this->continueAfterTEARDOWN);
```

Comment:
`res` is not used in the scope. You can either start checking the return value, or (in case there is no intend for it) drop it completely